src/io/flutter/logging/FlutterTreeTableModel.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging;
+
+import com.intellij.ui.treeStructure.treetable.TreeTable;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.event.TableModelListener;
+import javax.swing.table.TableModel;
+
+class FlutterTreeTableModel implements TableModel {
```

Comment:
This is a `TableModel` it supports for `TableRowSorter`.

This class doesn't affect anything related to display the log.